profiles/seccomp/seccomp_default.go

```diff
@@ -630,6 +630,19 @@ func DefaultProfile() *types.Seccomp {
 				Caps: []string{"CAP_SYS_TTY_CONFIG"},
 			},
 		},
+		{
+			Names: []string{
+				"get_mempolicy",
+				"mbind",
+				"name_to_handle_at",
```

Comment:
`name_to_handle_at` is nothing to do with `CAP_SYS_NICE` it is gated by `CAP_DAC_READ_SEARCH` and there are other reasons for excluding it.

Reply:
@justincormack I am totally fine removing it. But in that case the documentation would need to be updated at https://docs.docker.com/engine/security/seccomp/. Search for `name_to_handle_at`, it is mentioned `Already gated by CAP_SYS_NICE.`.

Reply:
Opened docker/docker.github.io#6854